src/bomf/model/__init__.py

```diff
@@ -47,12 +46,12 @@ class BusinessObjectRelation:
     All these relation types are 1:1 relations between business partners and adresses, yet they all carry different
     meaning which we'd like to distinguish in our data.
     """
-    relation_part_a: Bo4eTyp = attrs.field()
+    relation_part_a: Bo4eTyp = Any
     """
     one Business Object or COM
     """
 
-    relation_part_b: Bo4eTyp = attrs.field()
+    relation_part_b: Bo4eTyp = Any
```

Comment:
in dieser klasse kannst du die Anys wieder wegmachen.
das problem ist, dass du mit `= Any` dem entsprechenden field als default **value** den **type** `Any` zuweist.